lib/pub_sub/tasks/debug.rake

```diff
@@ -29,11 +29,18 @@ namespace :pub_sub do
     desc 'List information about the queue subscriptions.'
     task subscriptions: :environment do
       puts 'Subscriptions: ', '----------'
+      puts "SERVICE\tSUBSCRIPTION\tPROTOCOL\tREGION"
       PubSub.config.regions.each do |region|
-        subs = Aws::SNS::Client.new(region: region).list_subscriptions.subscriptions
+        subs = []
+        next_token = ""
+        while true
```

Comment:
I favor `loop do` as well. also seems to be the standard based of the ruby community for infinite loops.